temba/orgs/migrations/0074_auto_20210630_1156.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.2.10 on 2021-06-30 11:56
+
+from django.db import migrations
+from django.utils.timezone import timedelta, now as tz_now
+
+
+def update_topup_expiration_to_last_50_years(apps, schema_editor):
+    db_alias = schema_editor.connection.alias
+    TopUp = apps.get_model("orgs", "TopUp")
+    TopUp.objects.using(db_alias).update(expires_on=(tz_now() + timedelta(days=(365 * 50))))
+
+
+def update_topup_expiration_to_last_1_year(apps, schema_editor):
+    db_alias = schema_editor.connection.alias
+    TopUp = apps.get_model("orgs", "TopUp")
+    TopUp.objects.using(db_alias).update(expires_on=(tz_now() + timedelta(days=365)))
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("orgs", "0073_merge_20201214_1542"),
+    ]
+
+    operations = [
+        migrations.RunPython(update_topup_expiration_to_last_50_years, update_topup_expiration_to_last_1_year)
```

Comment:
Quick question: should we have both `update_topup_expiration_to_last_50_years` and `update_topup_expiration_to_last_1_year` here?

Reply:
`update_topup_expiration_to_last_1_year ` it's method that will be executed on migration revert. We can't revert it to the previous value but able to decrease it to one year as we had in the previous version. I think it's a good practice to provide a reverse method too.

Reply:
Oh, got it. Makes sense.